mockServer/src/services/schema2code.js

```
@@ -10,104 +10,9 @@
  */
 
-import { pageService } from '../routes/main-routes'
-import appInfo from '../assets/json/appinfo.json'
-
 export default class Schema2CodeServcice {
-  constructor() {
-    this.blockHistories = appInfo.blockHistories
-    this.components = appInfo.materialHistory.components
-  }
-
+  constructor() {}
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)